tedadhelper/src/main/java/gun0912/tedadhelper/nativead/TedNativeAd.java

```diff
@@ -96,6 +96,61 @@ public TedNativeAd(ViewGroup itemView, Context context, String app_name, String
         initView();
     }
 
+    public static class Builder {
+
+        // required
+        private ViewGroup containerView;
+        private Context context;
+        private String app_name;
+
+        // optional
+        String facebook_ad_key;
+        String admob_ad_key;
+        @TedAdHelper.ADMOB_NATIVE_AD_TYPE int admobNativeAdType;
+        TedAdHelper.ImageProvider imageProvider;
+
+        public Builder(ViewGroup containerView, Context context, String app_name) {
+            this.containerView = containerView;
+            this.context = context;
+            this.app_name = app_name;
+        }
+
+        public Builder setFacebook_ad_key(String facebook_ad_key) {
+            this.facebook_ad_key = facebook_ad_key;
+            return this;
+        }
+
+        public Builder setAdmob_ad_key(String admob_ad_key) {
+            this.admob_ad_key = admob_ad_key;
+            return this;
+        }
+
+        public Builder setAdmobNativeAdType(int admobNativeAdType) {
+            this.admobNativeAdType = admobNativeAdType;
+            return this;
+        }
+
+        public Builder setImageProvider(TedAdHelper.ImageProvider imageProvider) {
+            this.imageProvider = imageProvider;
+            return this;
+        }
+
+        public TedNativeAd build() {
```

Comment:
I think `create()` name is more better.
What do you think?

Reply:
I think ‘crete’ is better. But, ‘build’ is used by many source.